srcpkgs/mkbootimg/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'mkbootimg'
+pkgname=mkbootimg
+version=2021.04.27
+revision=1
+build_style=meta
+short_desc="Android mkbootimg + unpackbootimg, updated osm0sis fork"
+maintainer="Jami Kettunen <jami.kettunen@protonmail.com>"
+license="BSD-3-Clause"
+homepage="https://github.com/osm0sis/mkbootimg"
+distfiles="${homepage}/archive/refs/tags/${version}.tar.gz"
+checksum="a77a2c1a12804ea02cb808c08d80901ff7439d6184fbaa15726fcac4e167303d"
+
+do_build() {
+	make CC="$CC" AR="$AR rc"
+}
+
+do_install() {
```

Comment:
You can still use gnu-makefile and override only do_build.

Reply:
I'll keep that in mind, there isn't an `install` target in the Makefile so I'll still keep the `vbin`'s there.